### PR TITLE
Stop playlist in phonegap

### PR DESCRIPTION
**Pull request by ceoaliongroo**
#46

### CODE REVIEW

File: app/scripts/controllers/playMarker.js [hunk 1]
```diff
@@ -21,10 +21,15 @@ angular.module('publicEducationApp')
     $scope.editMode = Marker.isPlayingAllMarkers();
     $scope.actualPage = $location.absUrl();
 
-    $scope.PlayingAllMarkers = function() {
+    $scope.stopPlaying = function() {
+      // Stop the cycle, if it was enabled.
       Marker.setPlayingAllMarkers(false);
+
+      $scope.stopPlay = true;
```
Comment (amitaibu) [inline app/scripts/controllers/playMarker.js]: Lets change the property to `stopPlaying`

File: app/scripts/controllers/playMarker.js [hunk 2]
```diff
@@ -21,10 +21,15 @@ angular.module('publicEducationApp')
     $scope.editMode = Marker.isPlayingAllMarkers();
     $scope.actualPage = $location.absUrl();
 
-    $scope.PlayingAllMarkers = function() {
+    $scope.closePlaymode = function() {
```
Comment (amitaibu) [inline app/scripts/controllers/playMarker.js]: I liked `stopPlaying()` better.

Comment (ceoaliongroo) [inline app/scripts/controllers/playMarker.js]: stopPlaying is using for the property. stopPlayingAndCycle()

**Comment by amitaibu** [comment]
Merged.

**Comment by amitaibu** [comment]
Reverted.

**Pull request closed without merging**